src/components/FormTypes.tsx

```diff
@@ -0,0 +1,52 @@
+import React, { FunctionComponent, ReactNode } from "react";
```

Comment:
On these components if we remove the `pageText` input from the `BeaconsForm` and add it as a prop to each of the headings and put the text after the heading for each component